src/MicrosoftAuthFlow.js

```diff
@@ -32,6 +32,7 @@ class MicrosoftAuthFlow {
     this.options = options
     this.initTokenManagers(username, cache)
     this.codeCallback = codeCallback
+    this.options.skipTitleAuth = options.skipTitleAuth || !options.authTitle
```

Comment:
This seems like funny logic to me. Doesn’t seem like an ideal solution

Reply:
This was to prevent it being a breaking change as previously the condition for doing title auth was done only if authTitle was present which doesn't Work when supplied a client ID that cannot do title auth.